### PR TITLE
Cleaning up CI

### PR DESCRIPTION
**Pull request by jwoertink**
We need to be able to catch any nightly changes in Crystal against the latest Lucky. Also testing that edge Lucky works on the latest Crystal.

Cleaned up the check_format since we don't really need to test that it formats on all the different Crystal versions. As long as it's clean on the latest, that should be fine enough.

### CODE REVIEW

**Review by matthewmcgarvey — APPROVED** [review]
We might also consider a periodic run of the edge one since Lucky will be updated more than here

**Comment by jwoertink** [comment]
I like that idea. Any clue how I can set the cron to only run on a specific version?

**Comment by jwoertink** [comment]
ok, I think just having the second workflow will work

File: .github/workflows/weekly.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+name: Lucky CLI Weekly CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+  schedule:
+    - cron: "0 1 * * MON"
```
Comment (jwoertink) [inline .github/workflows/weekly.yml:8]: Run every monday at 1am
